### ISSUE TITLE
Update to quiz check

### ISSUE THREAD

**Issue by carriewright11** [jhudsl/ottrpal#117]
Noticed that we could add this to our quiz formatting checks... 

- Checking for space between quiz question instructions and question:

https://github.com/fhdsl/AI_for_software_Quizzes/pull/4

- Checking for question mark at the beginning of the question:

https://github.com/fhdsl/Ethical_Data_Handling_for_Cancer_Research_Quizzes/pull/10

- Checking for space after the question before the options:

https://github.com/fhdsl/Ethical_Data_Handling_for_Cancer_Research_Quizzes/pull/6/commits/ce4e4d1d42fbac311ab6f4370646ebad1cb8a015

I think this was a problem too (I am just so used to using ** for bold text but maybe it is a silly error to include haha): 
https://github.com/fhdsl/Ethical_Data_Handling_for_Cancer_Research_Quizzes/pull/11

**Comment by carriewright11** [comment]
@cansavvy don't know if you want this issue on the main ottr_template repo

**Comment by cansavvy** [comment]
Yeah, I've generally kept all the issues on the same repo so as it make it simpler for tracking. Because sometimes its not clear the solution and what repo might be involved in creating that solution.
